compose/foundation/foundation/build.gradle

```
@@ -84,6 +84,15 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
     }
 
     kotlin {
+        macosX64 {
```

Can you please add comment here
```suggestion
        // linker-option need to run tests on macOS
        macosX64 {
```

---

Done